custom/reach/static/reach/js/reports/program_overview.js

```diff
@@ -0,0 +1,90 @@
+hqDefine("reach/js/reports/program_overview", [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    'reach/js/utils/reach_utils',
+], function (
+    $,
+    ko,
+    _,
+    initialPageData,
+    reachUtils
+) {
+    var indicatorModel = function (options) {
+        var self = {};
+
+        self.reachUtils = reachUtils.reachUtils();
+        self.indicator = options.indicator;
+        self.color = options.color;
+        self.format = options.format;
+        self.value = options.value;
+        self.total = options.total || 0;
+        self.pastMonthValue = options.past_month_value || 0;
+
+        self.isNumeric = function () {
+            return self.format === 'numeric';
+        };
+
+        self.isPercent = function () {
+            return self.format === 'percent';
+        };
+
+        self.percentFormat = function () {
+            var value = self.value * 100 / (self.total || 1);
+            return value.toFixed(2) + " %";
+        };
+
+        self.diffBetweenMonths = function () {
```

Comment:
I'm not sure that this is the correct way to compare previous month and current month numbers but i will check